.github/workflows/check-test-release.yml

```diff
@@ -153,12 +153,10 @@ jobs:
         echo ::set-output name=tag::${GITHUB_REF#refs/tags/}
         git log --pretty='format:%d%n- %s%n%b---' $(git tag --sort=v:refname | tail -n2 | head -n1)..HEAD > _CHANGES.md
```

Comment:
I believe you no longer need these lines (i.e. can delete the `name: Changelog` step)

Reply:
https://github.com/iterative/mlem/pull/500

Reply:
it was to extract the changelog from git commits (to help draft release notes).

It used to be in a separate (bash) step because the old way used an action (https://github.com/softprops/action-gh-release) rather than plain bash.